src/components/topbar/TryVueNodeBanner.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div
+    v-if="showVueNodesBannerRef"
+    class="pointer-events-auto w-full h-10 bg-gradient-to-r from-blue-600 to-blue-700 flex items-center justify-between px-4"
+  >
+    <div class="w-5 h-5"></div>
+    <div class="flex items-center">
+      <i class="icon-[lucide--sparkles]"></i>
+      <h5 class="pl-2">{{ $t('vueNodesBanner.message') }}</h5>
+      <Button
+        class="cursor-pointer bg-transparent rounded h-7 px-3 border border-white text-white ml-4 text-xs"
+        @click="handleTryItOut"
+      >
+        {{ $t('vueNodesBanner.tryItOut') }}
+      </Button>
+    </div>
+    <Button
+      class="cursor-pointer bg-transparent border-0 outline-0 grid place-items-center"
+      unstyled
+      @click="handleDismiss"
+    >
+      <i class="w-5 h-5 icon-[lucide--x]"></i>
+    </Button>
+  </div>
+</template>
+
+<script setup lang="ts">
+import Button from 'primevue/button'
+import { onMounted, ref } from 'vue'
+
+import { useSettingStore } from '@/platform/settings/settingStore'
+
+const STORAGE_KEY = 'vueNodesBannerDismissed'
+
+const settingStore = useSettingStore()
+const showVueNodesBannerRef = ref(false)
+
+const checkLocalStorage = (): boolean => {
+  try {
+    const value = localStorage.getItem(STORAGE_KEY)
```

Comment:
question: Could we use the [VueUse composable](https://vueuse.org/core/uselocalstorage/#uselocalstorage)?